src/main/java/com/github/lmiljak/dowertefence/commands/DTCommand.java

```diff
@@ -0,0 +1,25 @@
+package com.github.lmiljak.dowertefence.commands;
+
+import nl.taico.taeirlib.commands.CommandArgs;
+import nl.taico.taeirlib.commands.TCommand;
+
+/**
+ * Class for handling the /dt command and subcommands.
+ */
```

Comment:
"and all subcommands", does that mean that all subcommands need to be handled in the same class, or can I for example also make a "/dt mySubCommand" in another class?